Controller/comments.controller.js

```diff
@@ -2,35 +2,52 @@ const { checkExist } = require("../Models/checkExists");
 const {
   selectAllCommentsById,
   addCommentByArticleId,
+  removeCommentById,
 } = require("../Models/comments.models");
 
 exports.getArticleCommentsById = (req, res, next) => {
-  const articleId  = req.params.article_id;
+  const articleId = req.params.article_id;
   const { sort_by, order } = req.query;
 
-  const promises = [selectAllCommentsById(articleId, sort_by, order)]
+  const promises = [selectAllCommentsById(articleId, sort_by, order)];
 
-  if(articleId){
-    promises.push(checkExist("articles","article_id",articleId))
+  if (articleId) {
```

Comment:
As above, you are using a parametric endpoint, this will always exist. It may not be valid, but it will exist.